daemon/create.go

```diff
@@ -14,26 +14,35 @@ import (
 	"github.com/opencontainers/runc/libcontainer/label"
 )
 
+// ContainerCreate is the parameter set to ContainerCreate()
+type ContainerCreate struct {
+	Name            string
+	Config          *runconfig.Config
+	HostConfig      *runconfig.HostConfig
+	AdjustCPUShares bool
+	FromBuilder     bool
```

Comment:
So why in https://github.com/docker/docker/pull/16650/files does it iterate through volumes?